server/test/cg/group4/server/package-info.java

```diff
@@ -1,4 +0,0 @@
-/**
```

Comment:
Not the most urgent issue;
What is the reason behind removing these package-info.java's? Is it because otherwise Eclipse conflicts about it?